imagestreams/golang-centos7.json

```diff
@@ -0,0 +1,47 @@
+{
+    "apiVersion": "v1",
+    "kind": "ImageStream",
+    "metadata": {
+        "annotations": {
+            "openshift.io/display-name": "Go"
+        },
+        "name": "golang"
+    },
+    "spec": {
+        "tags": [
+            {
+                "annotations": {
+                    "description": "Build and run Go applications on CentOS 7. For more information about using this builder image, including OpenShift considerations, see https://github.com/sclorg/s2i-nodejs-container/blob/master/8/README.md.\n\nWARNING: By selecting this tag, your application will automatically update to use the latest version of Node.js available on OpenShift, including major versions updates.",
+                    "iconClass": "icon-go-gopher",
+                    "openshift.io/display-name": "Go (Latest)",
+                    "openshift.io/provider-display-name": "Red Hat, Inc.",
+                    "sampleRepo": "https://github.com/sspeiche/golang-ex.git",
+                    "supports": "golang",
+                    "tags": "builder,golang,go"
+                },
+                "from": {
+                    "kind": "ImageStreamTag",
+                    "name": "1.8"
+                },
+                "name": "latest"
```

Comment:
You're missing:
```

        "referencePolicy": {
          "type": "Local"
        }
```
here. (See [example](https://github.com/sclorg/s2i-ruby-container/blob/master/imagestreams/ruby-centos7.json#L27))